README.md

```diff
@@ -1,55 +1,31 @@
 [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![Build status](https://ci.appveyor.com/api/projects/status/f4lrwf0x1uclsort/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/bhom_ui/branch/master)
+[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/BHoM_UI/BHoM_UI.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=17&branchName=master)
```

Comment:
What's the definitionId for BHoM_UI ?

Reply:
`19` - full badge is: `[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/BHoM_UI/BHoM_UI.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=19&branchName=master)`

Reply:
To avoid asking the same question for each PR, how can I find the number for each repo ?